### PR TITLE
[CYS] Pattern Sidebar: Update pattern categories and copy

### LINKED ISSUE #48381: [CYS] Pattern Sidebar: Review pattern categories and copies
Currently, the pattern sidebar is missing text copy. We should add them: https://github.com/woocommerce/woocommerce/blob/b6b5e23f9e92c158a58fe15f55b358afb51310ad/plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts/#L6-L28

| Intro | About | Service | Testimonials |
|--------|--------|--------|--------|
| image|image|image| image | 

Intro — Welcome shoppers to your store with one of our introductory patterns.
Feature selling — Put the spotlight on one or more of your products or product categories.
About — Show your shoppers what’s special about your business.
Reviews — Encourage sales by sharing positive feedback from happy shoppers.
Newsletter — Keep in touch with your shoppers with a newsletter signup form.
Social media — Promote your social channels and give shoppers a way to see your latest products and news.
Services — Share information on any services that your business can provide.

Furthermore, we have to ensure the pattern categories that we want to render.

### PR DESCRIPTION
**Pull request by albarin**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/48381

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure you have the `WooCommerce beta tester` plugin installed.
2. Head over to `Tools > WCA Test Helper > Features`.
3. Make sure you see the `pattern-toolkit-full-composability` on the list.
4. Enable it.
5. Go to `wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`, **disable** `Allow usage of WooCommerce to be tracked`, and save.
6. On the same page, check the option again and Save.
7. Go to `wp-admin/tools.php?page=action-scheduler`, search for the `fetch_patterns` action, and make sure it was scheduled.
8. Click on `Run` under the action name if it is still pending and wait until is completed.
9. Go to `wp-admin/admin.php?page=wc-admin&customizing=true&path=%2Fcustomize-store%2Fassembler-hub%2Fhomepage`.
10. Click on each one of the categories and make sure they all have a title and a description and they match the following:
```
Intro — Welcome shoppers to your store with one of our introductory patterns.
Feature selling — Put the spotlight on one or more of your products or product categories.
About — Show your shoppers what’s special about your business.
Services — Share information on any services that your business can provide.
Reviews — Encourage sales by sharing positive feedback from happy shoppers.
Social media — Promote your social channels and give shoppers a way to see your latest products and news.
```

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [x] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
CYS - Update pattern categories and its descriptions.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts [hunk 1]
```diff
@@ -6,22 +6,52 @@ import { __ } from '@wordpress/i18n';
 // @todo: Implement the right copy for the pattern categories: https://github.com/woocommerce/woocommerce/issues/48381
```
Comment (gigitux) [inline plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts]: I would remove this comment!

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts [hunk 2]
```diff
@@ -6,22 +6,52 @@ import { __ } from '@wordpress/i18n';
...
-		label: __( 'testimonials', 'woocommerce' ),
-		description: __( 'testimonials', 'woocommerce' ),
+	'social-media': {
+		label: __( 'Social media', 'woocommerce' ),
+		description: __(
+			'Promote your social channels and give shoppers a way to see your latest products and news.',
+			'woocommerce'
+		),
 	},
+	// newsletter: {
```
Comment (gigitux) [inline plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts:49]: Is there a particular reason why we keep this in the code?

**Review by gigitux — APPROVED** [review]
I left some minor comments, but I'm pre-approving it! Thanks! 🙇 🚀

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts (hunk 2 above)
Comment (albarin) [reply plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts:49]: I think this is a category that we'll be adding eventually, so I thought about leaving it there for when it's added.

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts (hunk 1 above)
Comment (albarin) [reply plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts]: Thanks, fixed!

File: plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts (hunk 2 above)
Comment (gigitux) [reply plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/pattern-screen/categories.ts:49]: Ok, thanks! 🙇

**Pull request merged**